app/views/middleware_server/show.html.haml

```diff
@@ -4,6 +4,7 @@
   = render(:partial => "layouts/performance_async")
 - elsif @showtype == 'main'
   = render :partial => "layouts/textual_groups_generic"
+  = render :partial => 'jdr_reports_list' if @record.middleware_jdr_reports.any?
```

Comment:
Are there any other view rendering with conditionals? If so, that's ok, but if not, we can just add the conditional to the partial itself so the main view is only calling partials, not deciding on their behalf.